docs/config.rst

```diff
@@ -110,7 +110,20 @@ Core configuration
     The original MPD server only supports 10000 tracks in the tracklist. Some
     MPD clients will crash if this limit is exceeded.
 
-
+.. confval:: core/restore_state
```

Comment:
Best would be if we don't need a configuration value for the 'restore state at start' feature. Restore 'last' state is what I expect. Like my car CD player do. Or any (old style) FM radio.

But, ...
I am not sure how may errors, support request, it will generate. 
Maybe we need the configuration only for 'ass saving'...

What are your experiences with mopidy?